src/main/java/mate/academy/hibernate/relations/model/Actor.java

```diff
@@ -1,41 +1,32 @@
 package mate.academy.hibernate.relations.model;
 
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@Entity
+@NoArgsConstructor
+@AllArgsConstructor
 public class Actor implements Cloneable {
```

Comment:
let's explicitly set table name through `@Table` annotation (use plural) - for all entities